### PR TITLE
feat: ingesting analytics tags

### PR DESCRIPTION
**Pull request by chronark**
## Summary by CodeRabbit

- **New Features**
	- Introduced new jobs for ClickHouse migration in the deployment workflow.
	- Added support for tags in verification requests, enhancing data aggregation capabilities.
	- New tasks for resetting ClickHouse migrations and improved error handling in verification processes.
  
- **Bug Fixes**
	- Updated SQL queries to ensure accurate data retrieval and aggregation.

- **Documentation**
	- Added a new section on "Tags" in the API documentation for better user guidance.

- **Tests**
	- Enhanced testing coverage for tag functionality and materialized views in ClickHouse.

### CODE REVIEW

File: apps/docs/apis/features/analytics.mdx [hunk 1]
```diff
@@ -21,3 +21,61 @@ Our per key analytics give you a deep dive into each individual key, giving usag
...
+## Tags
+
+You can add tags to verification requests to aggregate or filter data when querying.
+
+For example you might want to add the path of your API as a tag: `path=/v1/my-path` and then later retrieve a breakdown of api key usage per unique path.
+
+Unkey does not parse tags in any special way.
+The only limitations are:
+- Tags are strings.
+- You can specify up to 10 tags.
```
Comment (ogzhanolguncu) [inline apps/docs/apis/features/analytics.mdx:35]: Maybe, we should say 10 tags per request to make it a bit clear?

**Comment by ogzhanolguncu** [comment]
Other than that 10 request part everything looks good to me. Reason for 10 request part was people might assume they are limited to 10 tag variants etc... Maybe I'm overthinking.

File: apps/docs/apis/features/analytics.mdx (hunk 1 above)
Comment (perkinsjr) [reply apps/docs/apis/features/analytics.mdx:35]: Agreed

**Review by perkinsjr — CHANGES_REQUESTED** [review]
See comment from oz in the docs

**Comment by chronark** [comment]
agreed, fixed it, let's merge

**Review by perkinsjr — APPROVED** [review]

**Review by ogzhanolguncu — APPROVED** [review]

**Pull request merged**